app/Models/Collection.php

```diff
@@ -111,6 +111,11 @@ public function banner(): ?string
         return $this->extra_attributes->get('banner');
     }
 
+    public function bannerUpdatedAt(): ?string
+    {
+        return $this->extra_attributes->get('banner_updated_at');
```

Comment:
I'd parse `banner_updated_at` right here, so we won't have to deal with parsing in multiple places.